packages/grpc-js/src/internal-channel.ts

```diff
@@ -684,7 +684,7 @@ export class InternalChannel {
     host: string,
     credentials: CallCredentials,
     deadline: Deadline
-  ): Call {
+  ): LoadBalancingCall | RetryingCall {
```

Comment:
For my knowledge, I often see either of these or `ResolvingCall` on my `grpc` traces. Curious what the difference between the 3 are and whether they indicate any specific pattern? `RetryingCall` is the one that's a bit confusing as we don't have gRPC retries enabled by default. Does this account for the `transparent` retries?

Reply:
Those classes do have tracers set up, specifically `load_balancing_call`, `retrying_call`, and `resolving_call`. You should see them if you have them enabled and you make any requests. All of those are part of the call stack that makes up the internal implementation of a call object. As you say, `RetryingCall` is used by default for transparent retries.